tests/subsys/llext/simple/src/object_ext.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2023 Intel Corporation.
```

Comment:
2024 has arrived, I just caught myself today doing this :calendar:

Reply:
Updated :clock3: